examples/preconditioned-solver/preconditioned-solver.cpp

```diff
@@ -40,8 +40,13 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <string>
 
 
+#define PERFSTUBS_USE_TIMERS
+#include "perfstubs_api/timer.h"
```

Comment:
Shouldn't we use some `ifdef` to keep the example working when tau isn't available? Or create a specific example with tau might be clearer for users as well